scipy/stats/tests/test_morestats.py

```diff
@@ -79,6 +79,14 @@ def test_expon(self):
             np.seterr(**olderr)
         assert_(A > crit[-1])
 
+    def test_gumbel(self):
+        rs = RandomState(1234567890)
```

Comment:
I think more than 6 digits for the random seed was causing an error on some machines (32bit?)